my-calendar/src/java/org/sakaiproject/widgets/mycalendar/ui/WidgetPage.html

```diff
@@ -174,13 +174,15 @@
 
 /**
  * Parse the event list and count the number of events that match the date
- * @param date from the datepicker, is in the format mm/dd/yyyy
+ * Note: the way this is written it will work even if the timezone on the Sakai
+ * end is not the same as the browser's.
```

Comment:
Right, the timezone set in Sakai doesn't match the setting in the user's machine. I noticed this because my machine is set to Eastern time and my preference in the Pepperdine site in Pacific time, 3 hours behind. 

On July 23, 2016 6:20:59 PM EDT, Steve Swinsburg <notifications@github.com> wrote:
>> @@ -174,13 +174,15 @@
>>  
>>  /**
>>   * Parse the event list and count the number of events that match
>the date
>> - * @param date from the datepicker, is in the format mm/dd/yyyy
>> + * Note: the way this is written it will work even if the timezone
>on the Sakai
>> + * end is not the same as the browser's.
>
>When you say 'Sakai end' what do you mean. The timezone a user has
>configured in their preferences? Or the timezone when the event was
>created?
>The timezone is passed into the page based on the Sakai Locale.
>
>---
>You are receiving this because you authored the thread.
>Reply to this email directly or view it on GitHub:
>https://github.com/steveswinsburg/sakai-widgets/pull/31/files/ea90ace7504bcd0689abea6cccc1129b9a76eab6#r71979098

-- 
Sent from my Android device with K-9 Mail. Please excuse my brevity.